scripts/migrations/migrations.sh

```diff
@@ -1,30 +1,37 @@
-function run_migrations() {
-  echo "Starting migrations..."
-  echo
+function version_greater_than() {
```

Comment:
`version_greater_or_equal ` is maybe a more clear name. I don't think we want this to run if the versions are equal though, do we?

Reply:
yes, you're right